mmv1/third_party/terraform/services/appengine/resource_app_engine_standard_app_version_test.go

```diff
@@ -259,6 +259,12 @@ resource "google_storage_bucket_object" "main" {
   name   = "main.py"
   bucket = google_storage_bucket.bucket.name
   source = "./test-fixtures/hello-world-flask/main.py"
+}
+
+resource "google_storage_bucket_iam_member" "build" {
```

Comment:
This is generally not how we manage iam permissions for tests because when the test finishes it will remove the permission, potentially causing other runs to fail.

Reply:
reverted.